library/components/TUrl.vue

```diff
@@ -28,16 +28,27 @@ export default {
     },
     includeUrlStyle: {
       type: Boolean,
-      default: true
+      default: true,
+    },
+    includeContextParam:{
+      type: Boolean,
+      default: false
     }
   },
   computed: {
     fullUrl() {
-      let allUrlParams = {...this.additionalUrlParams};
+      let allUrlParams = {};
+      const { 'context[org]': org, 'context[group]': group, ...restOfFilters } = this.getFiltersState;
```

Comment:
I think that sometimes we there is a `context[page]` in the url. This destructuring would not catch that and would mean `context[page]=issue_detail` (or whatever page name) could be in the generated URL. Would that be a problem?

Reply:
Nope! In this case the page is changing anyway. :D